src/Microsoft.AspNet.Server.Kestrel/KestrelServerInformation.cs

```diff
@@ -39,5 +39,34 @@ private static ICollection<string> GetAddresses(IConfiguration configuration)
 
             return addresses;
         }
+
+        private static int GetThreadCount(IConfiguration configuration)
+        {
+            var threadCountString = configuration["server.threadCount"];
```

Comment:
Adding more things to the config is just fine, but give kestrel specific entries like this a kestrel specific name. E.g. `kestrel.threadCount`.